---
title: "[TIMOB-2167] Android: unexpected results on XHR Error callback"
---
<table>
<tr><th>Type</th><td>Bug</td></tr>
<tr><th>Priority</th><td>High</td></tr>
<tr><th>Status</th><td>Closed</td></tr>
<tr><th>Resolution</th><td>Cannot Reproduce</td></tr>
<tr><th>Resolution Date</th><td>2014-02-11T17:44:51.000+0000</td></tr>
<tr><th>Affected Version/s</th><td>Release 3.0.0</td></tr>
<tr><th>Fix Version/s</th><td>n/a</td></tr>
<tr><th>Components</th><td>Android</td></tr>
<tr><th>Labels</th><td>core</td></tr>
<tr><th>Reporter</th><td>Thomas Huelbert</td></tr>
<tr><th>Assignee</th><td>Sunila</td></tr>
<tr><th>Created</th><td>2011-04-15T03:12:32.000+0000</td></tr>
<tr><th>Updated</th><td>2017-03-17T18:19:32.000+0000</td></tr>
</table>

<h3>Description</h3>

<div><p>1.5.0.70ce98 g2 running 2.2, not seen on 1.6</p>
<p>1.Platform&gt;xhr&gt;Error Call back<br>
resultsd: <a href="<a href="http://www.fre100.com" rel="nofollow" target="_blank">http://www.fre100.com</a>">www.fre100.com</a></p>
<p>expected: the g1 returned a bunch of data. Not sure if this is a
test issue on this platform, hence no regression tag.</p></div>

<h3>Comments</h3>

<ol>
<li>Junaid Younus 2012-08-22

   Tested on a Samsung Galaxy S2 using TiSDK 2.2.0v20120821095711, issue reproducible and still valid. </li>
<li>Sunila 2014-02-11

   I can't reproduce the issue, tried the following test case and the proper error is shown.
   
   var xhr = Titanium.Network.createHTTPClient();
   	xhr.onerror = function(e)
   	{
   		Ti.API.info("ERROR " + e.error);
   		alert(e.error);
   	};
   	xhr.onload = function()
   	{
   		Ti.API.info("SUCCESS");
   	};
   	// open the client and encode our URL
   	xhr.open('GET',"<a href="http://www.fre100.com" rel="nofollow" target="_blank">http://www.fre100.com</a>");
   	// send the data
   	xhr.send();</li>
<li>Lee Morris 2017-03-17

   Closing ticket as the issue cannot be reproduced.</li>
</ol>


<p><a href="/TIMOB/TIMOB-2167.json">JSON Source</a></p>